samples/secure-with-tls/README.md

```diff
@@ -0,0 +1,133 @@
+# Secure EventStoreDB with TLS
+
+- [Secure EventStoreDB with TLS](#secure-eventstoredb-with-tls)
+  - [Overview](#overview)
+    - [Certificates](#certificates)
+  - [Description](#description)
+  - [Running Sample](#running-sample)
+    - [1. Generate self-signed certificates](#1-generate-self-signed-certificates)
+    - [2. Run samples with Docker](#2-run-samples-with-docker)
+    - [3. Run samples locally (without Docker)](#3-run-samples-locally-without-docker)
+      - [3.1 Install certificate - Linux (Ubuntu, Debian)](#31-install-certificate---linux-ubuntu-debian)
+      - [3.2 Install certificate - Windows](#32-install-certificate---windows)
+      - [3.3. Run EventStoreDB node](#33-run-eventstoredb-node)
+      - [3.3 Run client application](#33-run-client-application)
+
+## Overview
+
+The sample shows how to run the .NET TCP client secured by TLS certificates.
+
+Read more in the docs:
+- [Security](https://developers.eventstore.com/server/v20/server/security/)
+- [Running EventStoreDB with `docker-compose`](https://developers.eventstore.com/server/v20/server/installation/docker.html#use-docker-compose)
+- [Event Store Certificate Generation CLI](https://github.com/EventStore/es-gencert-cli)
+
+It is essential for production use to configure EventStoreDB security features to prevent unauthorised access to your data.
```

Comment:
TLS prevents eavesdropping, not unauthorized access as we secure the communication channel only